app/src/main/java/com/example/superheroes/ui/theme/Theme.kt

```diff
@@ -117,7 +115,9 @@ fun SuperheroesTheme(
     val view = LocalView.current
     if (!view.isInEditMode) {
         SideEffect {
-            setUpEdgeToEdge(view, darkTheme)
+            val window = (view.context as Activity).window
+            window.statusBarColor = colorScheme.primary.toArgb()
```

Comment:
```suggestion
            window.statusBarColor = Color.Transparent.toArgb()
```
You need to make status bar color transparency.